app/data/compose/docker-compose-kubo.yml

```diff
@@ -8,6 +8,6 @@ services:
       - ./ipfs/import:/import
       - ./ipfs/data:/data/ipfs
     ports:
-      - "8080"
-      - "4001"
-      - "5001"
+      - "0.0.0.0:8080:8080"
+      - "0.0.0.0:4001:4001"
+      - "0.0.0.0:5001:5001"
```

Comment:
FWIW, I only needed 5001 to get the webui working. Oddly, this only works locally. On DO, I get this error and allowing CORS did not solve the issue. 
<img width="1116" alt="Screenshot 2023-04-13 at 05 46 05" src="https://user-images.githubusercontent.com/8304391/231722581-8e156532-9bff-4ada-8290-76a57a0cfcc4.png">

Reply:
ahh, just had to restart the container. Updating the README